Sources/Common/DataModel/Triangle/index.js

```diff
@@ -51,7 +51,7 @@ function vtkTriangle(publicAPI, model) {
 
   publicAPI.getCellDimension = () => 2;
   publicAPI.intersectWithLine = (p1, p2, tol, x, pcoords) => {
-    const outObj = { subId: 0, t: 0, intersect: -1 };
+    const outObj = { subId: 0, t: 0, intersect: -1, betweenPoints: false };
```

Comment:
why not compute `betweenPoints` in the line line intersection ?

Reply:
which btw, it is also missing `t` in `outObj`

Reply:
I just adapted the vtkTriangle for a quick fix because du vtkPlane.intersectWithLine has been updated a long time ago.
I need a quick fix for a client, I didn't investigate more about it.
The same question will come back. When you call "intersectWithLine" what do you expect in output if the intersection is between point or not ?